site/app/views/forum/ForumThreadView.php

```diff
@@ -1137,6 +1140,11 @@ public function createPost(array $thread, array $post, $unviewed_posts, $first,
 
         $post_user_info = [];
 
+        $is_OP = false;
+        if ($post["author_user_id"] == $first_post['author_user_id']) {
+            $is_OP = true;
+        }
+
```

Comment:
This can be cut. You can just assign `$post_user_info["is_OP"]` below to `($post["author_user_id"] === $first_post['author_user_id'])`

Reply:
Implemented just now, I cut out the code block involving $is_OP.

Reply:
Awesome. The PHP lint test can be fixed by changing the '==' to '===', as [loose comparison](https://www.copterlabs.com/strict-vs-loose-comparisons-in-php/) is not allowed by PHPstan. Once that's done I'll approve.